### PR TITLE
Maintain repository

### PR DESCRIPTION
**Pull request by jenniferboedker**
Clean up unused workflows, add new ones for more automation. Also adjusts the documentation to have the same structure as the other repositories.

### CODE REVIEW

File: .github/pr-labels.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+feature: ['feature/*', 'feat/*']
+fix: ['fix/*', 'hotfix']
+chore: ['chore/*', 'documentation/*', 'docs/*', 'ci/*', 'refactor/*']
```
Comment (KochTobi) [inline .github/pr-labels.yml:3]:
```suggestion
chore: ['chore/*', 'documentation/*', 'docs/*', 'ci/*', 'refactor/*']

```

File: .github/workflows/nexus-publish-snapshots.yml [hunk 2]
```diff
@@ -40,4 +46,3 @@ jobs:
         env:
           MAVEN_REPO_USERNAME: ${{ secrets.NEXUS_USERNAME }}
           MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
```
Comment (KochTobi) [inline .github/workflows/nexus-publish-snapshots.yml:48]:
```suggestion
          MAVEN_REPO_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
          
```
You can configure your IDE to always end files this way

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,43 +0,0 @@
-# Changelog for new releases
```
Comment (KochTobi) [inline CHANGELOG.md:1]: I think we must not delete the old changelog. The changes were not always in the release notes AFAIK.

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018, QBiC.
+Copyright (c) 2022, Universit of Tübingen.
```
Comment (KochTobi) [inline LICENSE:3]:
```suggestion
Copyright (c) 2018 - 2022, Universit of Tübingen.
```

File: README.rst [hunk 5]
```diff
@@ -82,9 +67,36 @@ Finding QBiC service instances from within an application is as easy as this:
...
+.. |codeql| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
+    :alt: CodeQl Status
+
+.. |license| image:: https://img.shields.io/github/license/qbicsoftware/core-utils-lib
+    :target: https://img.shields.io/github/license/qbicsoftware/core-utils-lib
+    :alt: Project Licence
+
+.. |release| image:: https://img.shields.io/github/v/release/qbicsoftware/core-utils-lib.svg?include_prereleases
+    :target: https://github.com/qbicsoftware/core-utils-lib/release
```
Comment (KochTobi) [inline README.rst:95]:
```suggestion
    :target: https://github.com/qbicsoftware/core-utils-lib/releases
```

File: README.rst [hunk 6]
```diff
@@ -82,9 +67,36 @@ Finding QBiC service instances from within an application is as easy as this:
...
+.. |maven-test| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/Run%20Maven%20Tests/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/Run%20Maven%20Tests/badge.svg
+    :alt: Github Workflow Tests Status
+
+.. |codeql| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
+    :alt: CodeQl Status
+
+.. |license| image:: https://img.shields.io/github/license/qbicsoftware/core-utils-lib
+    :target: https://img.shields.io/github/license/qbicsoftware/core-utils-lib
```
Comment (KochTobi) [inline README.rst:91]:
```suggestion
    :target: https://github.com/qbicsoftware/core-utils-lib/blob/master/LICENSE
```

File: README.rst [hunk 7]
```diff
@@ -82,9 +67,36 @@ Finding QBiC service instances from within an application is as easy as this:
...
 -------
 
-This project was created with qube_.
+This work is licensed under the `MIT license <https://mit-license.org/>`_.
+
+**Note**: This work uses the `Micronaut Framework <https://github.com/micronaut-projects>`_ and derivatives from the Micronaut framework family, its which are licensed under `Apache 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`_.
+
+
+.. |maven-build| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/Build%20Maven%20Package/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/Build%20Maven%20Package/badge.svg
```
Comment (KochTobi) [inline README.rst:79]:
```suggestion
    :target: https://github.com/qbicsoftware/core-utils-lib/actions/workflows/build_package.yml
```

File: README.rst [hunk 8]
```diff
@@ -82,9 +67,36 @@ Finding QBiC service instances from within an application is as easy as this:
...
+    :alt: Github Workflow Build Maven Package Status
+
+.. |maven-test| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/Run%20Maven%20Tests/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/Run%20Maven%20Tests/badge.svg
```
Comment (KochTobi) [inline README.rst:83]:
```suggestion
    :target: https://github.com/qbicsoftware/core-utils-lib/actions/workflows/run_tests.yml
```

File: README.rst [hunk 9]
```diff
@@ -82,9 +67,36 @@ Finding QBiC service instances from within an application is as easy as this:
...
+    :alt: Github Workflow Tests Status
+
+.. |codeql| image:: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
+    :target: https://github.com/qbicsoftware/core-utils-lib/workflows/CodeQL/badge.svg
```
Comment (KochTobi) [inline README.rst:87]:
```suggestion
    :target: https://github.com/qbicsoftware/core-utils-lib/actions/workflows/codeql-analysis.yml
```

**Review by KochTobi — CHANGES_REQUESTED** [review]
Hi jenny
looks good the links in the readme are not correct

File: CHANGELOG.md (hunk 3 above)
Comment (jenniferboedker) [reply CHANGELOG.md:1]: I checked and all the information is also contained in the rst

File: LICENSE [hunk 10]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018, QBiC.
+Copyright (c) 2018 - 2022, Universit of Tübingen.
```
Comment (KochTobi) [inline LICENSE:3]:
```suggestion
Copyright (c) 2018 - 2022, University of Tübingen.
```

**Review by KochTobi — APPROVED** [review]
LGTM nice job!

**Pull request merged**